content/90.back-matter.md

```diff
@@ -30,10 +30,16 @@ The plot on the right (B) is a bar chart of token frequency appearing in PMC and
 ](https://raw.githubusercontent.com/greenelab/annorxiver/master/biorxiv/corpora_comparison/output/figures/pmc_vs_reference.png){#fig:pmc_v_reference tag="S3"}
 -->
 
+![
+**A.** Preprints are closer in document embedding space to their corresponding peer-reviewed publication than they are to random papers published in the same journal.
+**B.** Both the full-text and abstract only distributions are centered near the bottom left of the graph, signifying that the abstract only distance positively correlates with full-text distance.
```

Comment:
I am not sure this is necessarily true. You could have no relationship but still have things in the bottom left of the graph if you had distant outliers (which might be what's happening). I'm not sure if you need this panel to address reviewer concerns. If you don't, I'd suggest dropping it in the interest of time or using a quantitative test.